main.tex

```diff
@@ -293,19 +291,20 @@ \section{Development history and status}
 contributors.
 
 Some contributions have come as direct institutional support.  The ESA GAIA and
-ESASky modules were provided by developers working for ESA.  Meanwhile, the
+ESASky modules were provided by developers working for ESA\@.  Meanwhile, the
```

Comment:
what does `\@` do?

Reply:
\@ says that the period is at the end of the sentence. It is only needed when the final character before the period is an upper case character.